### PR TITLE
Complete check-in #2 questionnaire

### PR DESCRIPTION
**Pull request by ryanflach**
Add feedback for second check-in. 

@allindow Please review =) Left the section on the stretch goal as pending - we can adjust that section and next steps based on the initial progress you're able to make on collision detection this evening.

### CODE REVIEW

File: 1605/module_4_assignments/gametime/rf_al_game_time/ryan_angela_check_in_2.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+## Instructor Goal Completion
+
+### Goal 1: Placeholder obstacles (no collision detection)
+
+  - Provide a screenshot or gif or link to code for this goal
+  ![Screenshot](http://i.imgur.com/8g8TzU5.jpg)
+  - Did you deviate from the instructions? Tell us about it.
+    * No, we stuck to what was originally discussed.
+  - Questions, Comments, Concerns?
+    * We still have some concerns regarding performance, but these issues are not consistent.
```
Comment (allindow) [inline 1605/module_4_assignments/gametime/rf_al_game_time/ryan_angela_check_in_2.md]: @ryanflach do you mean the choppiness of the background?

File: 1605/module_4_assignments/gametime/rf_al_game_time/ryan_angela_check_in_2.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  * Pending
+- Did you deviate from the instructions? Tell us about it.
+  * Pending
+- Questions, Comments, Concerns?
+  * Same concern as Goal 2.
+
+## Next Steps?
+
+- What do you feel the next steps should be?
+  * Implement game start and game end functionality
```
Comment (allindow) [inline 1605/module_4_assignments/gametime/rf_al_game_time/ryan_angela_check_in_2.md]: How about a...not sure what you would call it? Welcome screen giving a quick definition of a jeepney and showing the different obstacles/bonuses with what they're called. Or would that come after clicking start?

File: 1605/module_4_assignments/gametime/rf_al_game_time/ryan_angela_check_in_2.md (hunk 1 above)
Comment (ryanflach) [inline 1605/module_4_assignments/gametime/rf_al_game_time/ryan_angela_check_in_2.md]: @allindow Yes - were you experiencing this as well? I didn't notice it this morning.

File: 1605/module_4_assignments/gametime/rf_al_game_time/ryan_angela_check_in_2.md (hunk 2 above)
Comment (ryanflach) [inline 1605/module_4_assignments/gametime/rf_al_game_time/ryan_angela_check_in_2.md]: I like that idea. Our game instructions will be very simple, so we should be able to add some background information on the screen where they would click start.

**Comment by ryanflach** [comment]
@allindow I've updated and added to the stretch goal based on your completion of that feature. Want to give it a final look over before I remove the [WIP] tag from the PR?

**Comment by allindow** [comment]
This looks good

**Pull request merged**